r2dbc-mysql/src/main/java/JasyncMetadata.kt

```diff
@@ -0,0 +1,65 @@
+package com.github.jasync.r2dbc.mysql
+
+import com.github.jasync.sql.db.ResultSet
+import io.r2dbc.spi.ColumnMetadata
+import io.r2dbc.spi.RowMetadata
+import java.util.*
+
+
+class JasyncMetadata(rows: ResultSet) : RowMetadata {
```

Comment:
Fyi, relates to r2dbc/r2dbc-spi#56

Reply:
I am not sure if that is what you mean but in the driver we have col names: https://github.com/jasync-sql/jasync-sql/blob/dde3f6afeaa325189ed0467bfd1342bdce9adde9/db-async-common/src/main/java/com/github/jasync/sql/db/ResultSet.kt#L19

Reply:
We will add the column names shortcut as additional SPI method, so this is just a heads-up.